pkg/cmd/roachtest/tests/restore.go

```diff
@@ -660,6 +660,212 @@ func registerRestore(r registry.Registry) {
 			require.NotEqual(t, 3, maxPauses, "the job should have paused at least once")
 		},
 	})
+
+	for _, sp := range []restoreSpecs{
+		{
+			name:     "restore/nodes=4",
+			hardware: makeHardwareSpecs(hardwareSpecs{}),
+			backup:   makeBackupSpecs(backupSpecs{}),
+			timeout:  2 * time.Hour,
+		},
+		{
+			name:     "restore/gce",
```

Comment:
mmm, how do I know the number of nodes and cpus and cloud provider etc of the default?
why not keep the existing naming convention where you know roughly everything from the name of the test? I think it's easier to read and also being consistent with other roachtests makes more sense.
that said.. maybe I'm missing something?

Reply:
take a look at `makeHardwareSpecs()` and `makeBackupSpecs()` in restore.go

I like my design better because there are currently 4 `backupSpecs` and 4 `hardwareSpecs`, and I think it would be an eyesore to respecify each spec for each test. Further, it would be less clear which tests were using the default specs.